polars/polars-lazy/src/tests/tpch.rs

```diff
@@ -56,8 +56,7 @@ fn test_q2() -> PolarsResult<()> {
         .inner_join(nation(), "s_nationkey", "n_nationkey")
         .inner_join(region(), "n_regionkey", "r_regionkey")
         .filter(col("p_size").eq(15))
-        .filter(col("p_type").str().ends_with("BRASS"));
-
+        .filter(col("p_type").str().ends_with("BRASS".into()));
```

Comment:
This should be `lit("BRASS".into::<String>())`

Reply:
Thanks,
This test does not run locally, do you know why ? `make test` is not enough ?

Reply:
It is in the integration tests. Can you run those?

Reply:
Ok, it is because I was running testing only in polars/py-polars, and I should run them in polars/polars right ?